package.json

```diff
@@ -93,19 +94,19 @@
     "@react-spring/web": "^9.7.3",
     "@store-unit/react": "^1.0.4",
     "@tanstack/react-query": "^4.35.3",
-    "@tanstack/react-virtual": "^3.0.0-beta.54",
+    "@tanstack/react-virtual": "^3.2.0",
     "@walletconnect/jsonrpc-provider": "^1.0.13",
     "@walletconnect/jsonrpc-utils": "^1.0.8",
     "@zeriontech/components": "^1.0.1",
     "@zeriontech/hardware-wallet-connection": "^1.0.11",
-    "@zeriontech/transactions": "^0.0.12",
+    "@zeriontech/transactions": "0.0.13",
```

Comment:
why exact version?

Reply:
Just a habit to mention exact version. Will fix

Reply:
Fixed